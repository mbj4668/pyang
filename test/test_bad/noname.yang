module { // error: expected module name
}
