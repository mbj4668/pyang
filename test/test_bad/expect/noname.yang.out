noname.yang:1: warning: WBAD_MODULE_NAME
noname.yang:1: error: EXPECTED_ARGUMENT
